crates/ra_flycheck/src/lib.rs

```diff
@@ -300,15 +266,11 @@ fn run_cargo(
     // erroneus output.
     let stdout = BufReader::new(child.stdout.take().unwrap());
     let mut read_at_least_one_message = false;
-
-    for line in stdout.lines() {
```

Comment:
This might cause a regression with regards to how we handle junk output from stdout usage in build scripts, proc macros, etc, unless the issue has been fixed upstream.

Reply:
Ahh, I see oli-obk/cargo_metadata#106 should address this issue! Awesome